media/js/app/projects/selection_assignment_edit.js

```diff
@@ -36,23 +35,32 @@
                 minDate: 0,
                 dateFormat: 'mm/dd/yy'
             });
+
+            // Setup the media display window.
+            this.citationView = new CitationView();
+            this.citationView.init({
+                'default_target': 'asset-workspace-videoclipbox',
+                'presentation': 'medium',
+                'clipform': false,
+                'autoplay': false
+            });
+
+            jQuery(window).trigger('resize');
         },
         validate: function(pageNo) {
             var q;
             if (pageNo === 1) {
-                return jQuery(this.el).find('input[name="title"]')
-                                      .val()
-                                      .length > 0;
+                return jQuery('input[name="item"]').val() !== '';
             } else if (pageNo === 2) {
-                return tinyMCE.activeEditor.getContent().length > 0;
+                var title = jQuery(this.el).find('input[name="title"]').val();
```

Comment:
just a minor thing, but you can use `this.$el` instead of initializing `jQuery(this.el)` each time. http://backbonejs.org/#View-$el